routes.js

```diff
@@ -0,0 +1,63 @@
+var restify = require('restify');
+var Todo = require('./models/todo')(Todo);
+
+module.exports = (server) => {
+    var server = restify.createServer();
+    server.use(restify.bodyParser());
+
+    //Test 1: Create a Todo
+    server.post('/todos', (req, res, next) => {
+        Todo.create({
+            title: req.body.title,
+            completed: false
+        }).then((rows) => {
+            res.status(201);
+            res.charSet('utf-8');
+            res.send(rows.dataValues);
```

Comment:
You should stop thinking in terms of database, these aren't supposed to be rows that you access data values from - it's an ORM and generates objects. Todo object instances, not row object literals.